docs/src/examples/matrix-inversion-manual.jl

```diff
@@ -143,9 +143,11 @@ MOI.set(
     model,
     DiffOpt.ForwardInConstraint(),
     cons[1],
-    0.0 * index(x[1]) - 1.0,
+    0.0 * index(x[1]) - 1.0,  # the tangent of the ConstraintFunction, i.e., ∂(Gx - h)/∂h = -1
 )
 
+# Note that `0.0 * index(x[1])` is used to make its type `typeof(0.0 * index(x[1]) - 1.0) <: MOI.AbstractScalarFunction`.
```

Comment:
```suggestion
# Note that `0.0 * index(x[1])` is used to make its type `typeof(0.0 * index(x[1]) - 1.0) <: MOI.AbstractScalarFunction`.
# To indicate different direction to get directional derivative, users should replace `0.0 * index(x[1]) - 1.0` as the form of `A*x - b`, where `A` and `b` correspond to the elements of direction vectors along `G` and `h` axes, respectively.
```

Reply:
I would avoid using A and b, since they are used to describe equality constraints.
maybe `dG * x - dh`?

Reply:
IMO, it may also be confusing as in #174 we will finally get `dx/dy` where `y` denotes perturbation itself.
That is, if we use the notation as `dG` and `dh`, we will get something like `dx/ddh`.

However, I agree that still `dG` and `dh` are intuitive and convenient.

Reply:
Forward mode is usually simpler to understand. dG and dh also follow (closely) the notation of the paper OptNet

Reply:
Your comment is reflected.